tests/staticfiles_tests/project/documents/cached/module.js

```diff
@@ -0,0 +1,5 @@
+import rootModule from "/static/absolute_root.js"
+import testModule from "./test.js"
+import relativeModule from "../nested/js/nested.js"
+
+const dynamicImport = import("./test.js")
```

Comment:
Can we add tests for imports with curly brackets, `*`-imports, and multiline imports? e.g.
```javascript
import {function1, function2} from './test.js';
import {function3 as newFunction3} from './test.js';
import {function4 as newFunction4,
        function5 as newFunction5} from './test.js';
import * as NewModule from './test.js';
```

Reply:
Extended cases were included.